api/api/src/main/scala/uk/ac/wellcome/platform/api/rest/WorksController.scala

```diff
@@ -64,6 +64,8 @@ class WorksController(elasticsearchService: ElasticsearchService,
               workRedirect(work)
             case Right(Some(work: Work.Invisible[Indexed])) =>
               gone("This work has been deleted")
+            case Right(Some(work: Work.Deleted[Indexed])) =>
+              gone("This work has been deleted")
```

Comment:
Now we have this message, is it worth tweaking the one for Invisible?

"This work is not available"?

Reply:
I find that a bit weird cause the response it's still a gone 410 instead of a 404 not available. I don't think the difference `Invisible` and `Deleted` should be of interest from the API perspective, it's more a difference in treatment in the pipeline. If we had an API only internal model, I don't think we would need these two types